src/phoenix/experimental/evals/functions/generate.py

```diff
@@ -9,13 +9,18 @@
 logger = logging.getLogger(__name__)
 
 
+def _no_op_parser(response: str) -> Dict[str, str]:
+    return {"output": response}
+
+
 def llm_generate(
     dataframe: pd.DataFrame,
     template: Union[PromptTemplate, str],
     model: BaseEvalModel,
     system_instruction: Optional[str] = None,
     verbose: bool = False,
-) -> List[str]:
+    output_parser: Optional[Callable[[str], Dict[str, Any]]] = None,
```

Comment:
```suggestion
    output_parser: Optional[Callable[[str], Any]] = None,
```
Seems like the user might want to write all sorts of kinds of parsers.

Reply:
@axiomofjoy but we need to know what column to map it to - this at least forces you to declare the column. Or are you thinking of something different?